database/cassSeed.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const faker = require('faker');
+const { argv } = require('yargs');
+
+const start = argv.start || 1;
+const finish = argv.finish || 2;
+
+const filename1 = argv.output1 || 'cassPhotosByRoom.csv';
+const filename2 = argv.output2 || 'cassListByUser.csv';
+
+const stream1 = fs.createWriteStream(filename1);
+const stream2 = fs.createWriteStream(filename2);
+
+const photosByRoom = (roomId) => {
+  const randomNumPhotos = Math.floor(Math.random() * 17);
+  const room_id = roomId;
+  let str = '';
+
+  for (let i = 1; i < randomNumPhotos; i += 1) {
+    const photo_order = i;
+    const title = faker.lorem.words();
+    const address = faker.address.streetAddress();
+    const image_url = faker.image.imageUrl();
+    const image_description = faker.lorem.words();
+
+    str += `${room_id},${photo_order},${title},${address},${image_url},${image_description}\n`;
+  }
+
+  return `${str}`;
+};
+
+let listId = 1;
+
+const listByUser = (userId) => {
+  const randomNumList = Math.floor(Math.random() * 5);
```

Comment:
So, you are never getting 5 or 8, if it is how it should be, then no problem, but if not --> probably change random script.